Jasmine/Common/Helpers/Random.swift

```diff
@@ -10,10 +10,21 @@ enum Random {
     ///   - toInclusive: end of range, inclusive
     /// - Returns: a random number in the range
     static func integer(from: Int = 0, toInclusive: Int) -> Int {
-        assert(toInclusive > from, "`toInclusive` must be larger than `from` number")
+        assert(toInclusive > from, "`toInclusive` must be larger than `from`")
         return from + Int(arc4random_uniform(UInt32(toInclusive - from + 1)))
     }
 
+    /// Generates a random integer
+    ///
+    /// - Parameters:
+    ///   - from: start of range, inclusive
+    ///   - toExclusive: end of range, exclusive
+    /// - Returns: a random number in the range
+    static func integer(from: Int = 0, toExclusive: Int) -> Int {
+        assert(toExclusive > from + 1, "`toExclusive` must be larger than `from` by at least 2")
```

Comment:
Yeah then technically `toInclusive` is valid if it's == `from` too. I'm not sure if we want to allow this though, since it's more efficient to not use Random at all if the range of possible values is 1.